src/show/pages/private/Admin/index.jsx

```diff
@@ -13,76 +13,48 @@ import {
   addConstructor,
   getConstructors
 } from 'process/slices/constructorsSlice';
+import { updateProps } from 'process/slices/transientSlice';
 import { addEvent, getEvents } from 'process/slices/eventsSlice';
 import { addDriver, getDrivers } from 'process/slices/driversSlice';
 
 const Admin = () => {
-  const [createConstructorModal, setCreateConstructorModal] = useState(false);
-  const [createDriverModal, setCreateDriverModal] = useState(false);
   const [createEventModal, setCreateEventModal] = useState(false);
-  const [search, setSearch] = useState('');
-  const [driver, setDriver] = useState({
-    constructor_id: null,
-    first_name: '',
-    last_name: '',
-    description: '',
-    avatar_url: '',
-    race_number: '',
-    nickname: '',
-    birth_date: '',
-    nationality: '',
-    height: '',
-    points: ''
-  });
-  const [constructor, setConstructor] = useState({
-    name: '',
-    description: '',
-    logo_url: '',
-    cover_url: '',
-    first_apparence: '',
-    origins: '',
-    titles: '',
-    points: '',
-    car_url: ''
-  });
-  const [event, setEvent] = useState({
-    name: '',
-    city: '',
-    cover_url: '',
-    description: '',
-    country: '',
-    date: ''
-  });
+  const [createDriverModal, setCreateDriverModal] = useState(false);
+  const [createConstructorModal, setCreateConstructorModal] = useState(false);
 
   const dispatch = useDispatch();
 
+  const transient = useSelector(state => state.transient);
   const events = useSelector(state => state.events.events);
   const drivers = useSelector(state => state.drivers.drivers);
+  const searchQuery = useSelector(state => state.transient.searchQuery);
```

Comment:
```suggestion
  const searchQuery = transient.searchQuery;
```

You already have all the transient state, just put what you need in a constant instead of using this hook again (useSelector - hook)

Reply:
done